frontend/src/components/pages/DecryptPassword/DecryptPassword.js

```diff
@@ -73,6 +73,14 @@ const DecryptPassword = () => {
     setStats(computeDecryptedDataStats(decryptedData));
   }, [decryptedData]);
 
+  useEffect(() => {
```

Comment:
Why is this separate block required? This whole logic can be moved to previous useEffect with an else if case of when length > 0.

Reply:
I have  made the changes and squashed the commits into one single commit.